server/endpoint/endpoint.go

```diff
@@ -36,24 +30,27 @@ func New(config Config) (*Endpoint, error) {
 
 	var healthzEndpoint *healthz.Endpoint
 	{
-		healthzConfig := healthz.DefaultConfig()
-		healthzConfig.Logger = config.Logger
-		healthzConfig.Services = []healthzservice.Service{
-			config.Service.Healthz.K8s,
-			config.Service.Healthz.Vault,
+		c := healthz.Config{
+			Logger: config.Logger,
+			Services: []healthzservice.Service{
+				config.Service.Healthz.K8s,
```

Comment:
This is still checking against k8s API. Weren't we supposed to drop those as well?

Reply:
Uncertain. The postmortem and the issue were only about Vault dependency. We did not have huge problems with the k8s healthcheck so far. We should maybe think about the health checking practises in general and derive some guidelines we can apply so we get it straight once and for all.

Reply:
Roger that.